src/test/bpmtest.cpp

```diff
@@ -0,0 +1,17 @@
+#include <gtest/gtest.h>
+
+#include <QDebug>
+
+#include "track/bpm.h"
+
+class BpmTest : public testing::Test {
+};
+
+TEST_F(BpmTest, TestBpmEquality) {
+    EXPECT_EQ(mixxx::Bpm(120), mixxx::Bpm(120));
+    EXPECT_EQ(mixxx::Bpm(120), mixxx::Bpm(60) * 2);
+    EXPECT_EQ(mixxx::Bpm(120), mixxx::Bpm(240) / 2);
+
+    EXPECT_EQ(mixxx::Bpm(), mixxx::Bpm(mixxx::Bpm::kValueUndefined));
+    EXPECT_EQ(mixxx::Bpm(), mixxx::Bpm(0.0));
```

Comment:
This last test is confusing. We could test that `mixxx::Bpm::kValueUndefined` equals `0.0` but that doesn't really make any sense to me. Otherwise we would need to test the value of each and every constant.

Reply:
The point is that all invalid values should be equal to another. But it was actually not working, fixed that.